### ISSUE TITLE
Close button not showing for <ebay-page-notice>

### ISSUE THREAD

**Issue by ts-xie** [eBay/ebayui-core#2179]
# Bug Report

## Description

See thread: https://ebay-eng.slack.com/archives/CK8AX1L49/p1716581865060279

> a11y-dismiss-text doesn't get transformed to a11yDismissText since it's not included in marko-tag.json in eBayUI. This is a bug that should be fixed, but if you want a quick fix you can switch to camelCase from kebab-case for that specific attribute

Also might need to fix the example code should show close button when a11yDismissText is absent: https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-section-notice/examples/with-dismiss.marko

## Workaround

Use camelCase, ie, `a11yDismissText="close"`

## Screenshots

See screenshots in slack thread
<img width="1376" alt="Screenshot 2024-05-24 at 1 23 03 PM" src="https://github.com/eBay/ebayui-core/assets/8977441/08be945d-a731-4c61-bd2c-23f8d6a84f97">
